pkgs/development/python-modules/color-operations/default.nix

```
@@ -1,35 +1,40 @@
 {
   lib,
   buildPythonPackage,
-  fetchFromGitHub,
-  pytestCheckHook,
   pythonOlder,
+  fetchFromGitHub,
 
-  colormath,
+  # build-system
   cython,
-  oldest-supported-numpy,
   setuptools,
+
+  # dependencies
+  oldest-supported-numpy,
+
+  # checks
+  colormath,
+  pytestCheckHook,
 }:
 
 buildPythonPackage rec {
   pname = "color-operations";
-  version = "0.1.3";
+  version = "0.1.4";
   pyproject = true;
   disabled = pythonOlder "3.8";
 
   src = fetchFromGitHub {
     owner = "vincentsarago";
     repo = "color-operations";
-    rev = version;
-    hash = "sha256-KsrgilcNK2ufPKrhtGdf8mdlFzhsHB2jHN+WDlZqabc=";
+    rev = "refs/tags/${version}";
+    hash = "sha256-qqOTmVYD3VfjeVJtYvDQw+cxjcTsmqTYQNL1qMX+fL4=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
```

```suggestion
  build-system = [
    oldest-supported-numpy
```
Could you please add `oldest-supported-numpy` to `build-system`?
https://github.com/vincentsarago/color-operations/blob/d76a3464bef29e7c2ac29534f474f1f5229237d1/pyproject.toml#L1-L7